tests/test_setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+sudo apt-get -y update
+sudo apt-get install -y software-properties-common screen
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" -y
+sudo apt-get update -y
+sudo apt-get install docker-ce -y
+sudo usermod -aG docker $USER
+sudo curl -L "https://github.com/docker/compose/releases/download/1.23.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
docker-compose `1.23.1` has a specific feature to parallelize the build (helpful for faster build times). @darknight-007 
https://github.com/docker/compose/pull/6115